newa/__init__.py

```diff
@@ -301,6 +302,33 @@ class EventType(Enum):
     COMPOSE = 'compose'
 
 
+class SysArchType(Enum):
+    """ Available system architectures """
+
+    X86_64 = 'x86_64'
+    AARCH64 = 'aarch64'
+    S390X = 's390x'
+    PPC64LE = 'ppc64le'
+    MULTI = 'multi'
+
+    @classmethod
+    def architectures(cls: type[SysArchType],
+                      preset: Optional[list[SysArchType]] = None) -> list[SysArchType]:
+
+        _all = [
+            SysArchType('x86_64'),
+            SysArchType('s390x'),
+            SysArchType('aarch64'),
+            SysArchType('ppc64le')]
```

Comment:
To avoid the need to update this list when new architecture appears (and forgetting to do so...) you can use `[a.value for a in SysArchType.__members__.values() if a != SysArchType.MULTI]`

Reply:
Fixed.